roles/openshift_storage_glusterfs/templates/heketi-template.yml.j2

```diff
@@ -147,9 +145,7 @@ parameters:
   displayName: GlusterFS cluster name
   description: A unique name to identify this heketi service, useful for running multiple heketi instances
   value: glusterfs
-{% if glusterfs_is_native | bool %}
-- name: HEKETI_LVMWRAPPER
+- name: HEKETI_LVM_WRAPPER
   displayName: Wrapper for executing LVM commands
   description: Heketi can use a wrapper to execute LVM commands, i.e. run commands in the host namespace instead of in the Gluster container.
   value: "/usr/sbin/exec-on-host"
```

Comment:
Should we leave the value empty instead of this default?

Reply:
Ok, @phlogistonjohn explained this to me in a call. HEKETI_LVM_WRAPPER will default to value we want it to be in latest 3.11 OCS in Converged mode. For other setups, it can be tweaked if necessary.